src/main/java/org/openstreetmap/atlas/checks/validation/tag/MixedCaseNameCheck.java

```diff
@@ -127,6 +135,12 @@ public boolean validCheckForObject(final AtlasObject object)
                                 .contains(object.tag(ISOCountryTag.KEY).toUpperCase())
                         // And have a name tag
                         && Validators.hasValuesFor(object, NameTag.class)
+                        // Excluding names with following tags to reduce number of legit
+                        // MixedCaseNames
+                        && !Validators.hasValuesFor(object, BrandTag.class)
+                        && !Validators.hasValuesFor(object, ShopTag.class)
+                        && !Validators.hasValuesFor(object, AmenityTag.class)
+                        && !Validators.hasValuesFor(object, LeisureTag.class)
```

Comment:
this could be combined to `&& !Validators.hasValuesFor(object, BrandTag.class, ShopTag.class, AmenityTag.class, LeisureTag.class)`